components/Chips/examples/ChipsShapingExampleViewController.m

```diff
@@ -80,7 +67,8 @@ - (void)viewDidLoad {
   [_cornerSlider addTarget:self
                     action:@selector(cornerSliderChanged:)
           forControlEvents:UIControlEventValueChanged];
-  [MDCSliderColorThemer applySemanticColorScheme:_colorScheme toSlider:_cornerSlider];
+  [MDCSliderColorThemer applySemanticColorScheme:self.containerScheme.colorScheme
```

Comment:
`colorScheme` is nullable, but the themer only accepts `nonnull` arguments. Please use a conditional to provide a default value.